### ISSUE TITLE
Sort by size & delete multiple selection

### ISSUE THREAD

**Issue by abbas1511** [httptoolkit/feedback#83]
Here we cannot sort by size and/or delete multiple selection. Clear button will remove all the logs.

![image](https://user-images.githubusercontent.com/3536054/86461172-32c01c00-bd47-11ea-81fe-fe8355e845be.png)

---

_Does this affect you too? Click the +:smiley: icon in the top right and add a :+1: on issues to help decide where HTTP Toolkit goes next, or [go take a look & vote on the most popular ideas so far](https://github.com/httptoolkit/feedback/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)._

**Comment by pimterry** [comment]
Interesting, ok! I'm interested in this, but the UX and the details are tricky.

Why do you want to sort by size? It sounds like maybe you're trying to delete all the large requests or something, is that right? That makes sense if so, I'm just wondering if that might mean there are a other ways that this could be supported too (e.g. filtering + multi-select instead, so you could search for "size>100KB", select all, and delete from there)

**Comment by abbas1511** [comment]
yes, deletion is one of the reason and other would be to have a quick check on component size. Sorting would make it easy to focus on heavy files. Having a feature to filter+multi select for size would be a good start and can be fine tuned.
